docs/_docs/step-by-step/03-front-matter.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: step
+title: Front Matter
+position: 3
+---
+Front matter is a snippet of [YAML](http://yaml.org/) which sits between two
```

Comment:
can you uppercase `matter` in this document for consistency?

Reply:
I tried to match the usage in the docs but it's a little inconsistent. https://jekyllrb.com/docs/frontmatter/ uses front matter (lowercase) and Front Matter in headings. There's some places where it's frontmatter which should probably be renamed.

Reply:
I ended up going through all the docs making is consistent.

There were a few variations:

* frontmatter
* YAML Front Matter
* YAML front matter
* front-matter

I've changed it to "Front Matter" for headings, "front matter" everywhere else.

Reply:
Related: https://github.com/jekyll/jekyll/issues/2396#issuecomment-50858507